src/Microsoft.Identity.Client/ApiConfig/IAcquireTokenOnBehalfOfParameters.cs

```diff
@@ -37,6 +37,6 @@ internal interface IAcquireTokenOnBehalfOfParameters : IAcquireTokenCommonParame
 
         /// <summary>
         /// </summary>
-        bool WithOnBehalfOfCertificate { get; }
+        bool SendX5C { get; }
```

Comment:
>SendX5C [](start = 13, length = 7)

We should ensure the explanation of the usage of this one is clear. It's used for the subjectname + issuer feature allowing evo to roll certs with-out having to have both certs in the system when rolling